app/src/main/java/com/duckduckgo/app/bookmarks/service/SavedSitesExporter.kt

```diff
@@ -19,58 +19,63 @@ package com.duckduckgo.app.bookmarks.service
 import android.content.ContentResolver
 import android.net.Uri
 import com.duckduckgo.app.bookmarks.db.BookmarksDao
+import com.duckduckgo.app.bookmarks.db.FavoritesDao
 import com.duckduckgo.app.global.DefaultDispatcherProvider
 import com.duckduckgo.app.global.DispatcherProvider
 import kotlinx.coroutines.withContext
 import java.io.FileNotFoundException
 import java.io.FileOutputStream
 import java.io.IOException
 
-interface BookmarksExporter {
-    suspend fun export(uri: Uri): ExportBookmarksResult
+interface SavedSitesExporter {
+    suspend fun export(uri: Uri): ExportSavedSitesResult
 }
 
-sealed class ExportBookmarksResult {
-    object Success : ExportBookmarksResult()
-    data class Error(val exception: Exception) : ExportBookmarksResult()
-    object NoBookmarksExported : ExportBookmarksResult()
+sealed class ExportSavedSitesResult {
+    object Success : ExportSavedSitesResult()
+    data class Error(val exception: Exception) : ExportSavedSitesResult()
+    object NoSavedSitesExported : ExportSavedSitesResult()
 }
 
-class RealBookmarksExporter(
+class RealSavedSitesExporter(
     private val contentResolver: ContentResolver,
     private val bookmarksDao: BookmarksDao,
-    private val bookmarksParser: BookmarksParser,
+    private val favoritesDao: FavoritesDao,
+    private val savedSitesParser: SavedSitesParser,
     private val dispatcher: DispatcherProvider = DefaultDispatcherProvider()
-) : BookmarksExporter {
+) : SavedSitesExporter {
 
-    override suspend fun export(uri: Uri): ExportBookmarksResult {
+    override suspend fun export(uri: Uri): ExportSavedSitesResult {
         val bookmarks = withContext(dispatcher.io()) {
             bookmarksDao.getBookmarksSync()
         }
+        val favorites = withContext(dispatcher.io()) {
+            favoritesDao.favoritesSync()
```

Comment:
As commented above, I've created this same method for another usecase. however I exposed it in the repository. Will it makes sense to access through the repository instead of directly the dao?